src/V3AstNodeOther.h

```diff
@@ -2559,6 +2559,29 @@ class AstAlwaysPublic final : public AstNodeStmt {
     bool isJustOneBodyStmt() const { return stmtsp() && !stmtsp()->nextp(); }
     bool isFirstInMyListOfStatements(AstNode* n) const override { return n == stmtsp(); }
 };
+class AstAssertCtl final : public AstNodeStmt {
+    const VAssertCtlType m_ctlType;
+    string m_name;  // scope, kept under name show it on the graph
```

Comment:
Should the scope instead be handled similar to how AstDisplay uses AstScopeName?

Reply:
I think so. However, AstScopeName is resolved during V3Scope phase which is after V3Assert. I think I'll need to add new phase after V3Scope which will ensure correctness of AssertCtl and AssertCtlCheck scopes. This is also a reason why it fails with `-fno-inline`.